r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -148,13 +159,12 @@ test_that("Group by var on dataset", {
     input %>%
       group_by(some_grouping) %>%
       summarize(var = var(int, na.rm = FALSE)) %>%
-      arrange(some_grouping) %>%
       collect(),
     tbl
   )
 })
 
-test_that("n()", {
+test_that("Group by any/all", {
```

Comment:
This looks like it was introduced accidentally in a merge
```suggestion
test_that("n()", {
```